mongodump/mongodump_test.go

```diff
@@ -828,6 +828,44 @@ func TestMongoDumpBSONLongCollectionName(t *testing.T) {
 	})
 }
 
+func TestMongoDumpAtlasProxy(t *testing.T) {
```

Comment:
It'd be nice to use `testify` for new test methods. I've been trying to move this code away from `convey`.

Reply:
Should we wait until we have a longer term plan for the tools? The testing in general is a bit of a mess, and I'm not sure if its worth doing something like that till we have a plan.

Reply:
I've been using `testify` for new tests so there's already some code using it.